homeassistant/components/switchbot/humidifier.py

```diff
@@ -40,6 +40,7 @@ class SwitchBotHumidifier(SwitchbotSwitchedEntity, HumidifierEntity):
     _attr_available_modes = [MODE_NORMAL, MODE_AUTO]
     _device: switchbot.SwitchbotHumidifier
     _attr_min_humidity = 1
+    _attr_translation_key = "switchbot_humidifier"
```

Comment:
We don't have to prefix with the domain it in this case, translations are already scoped by nature.